static/css/api_key_settings.css

```
@@ -844,6 +844,62 @@ opacity: 0.95 !important;
+    font-weight: 500;
+    color: var(--text-primary, #333);
+}
+
+.disable-tts-toggle input[type="checkbox"] {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)